src/clr-angular/typography/_variables.typography.scss

```diff
@@ -13,9 +13,6 @@ $clr-app-font-color-primary: clr-getTextColor();
 // Usage: ../utils/_maps.clarity.scss
 $clr-altFont: $clr-font !default; //TODO: TBD. Alternative font for Clarity
 
-// Usage: ../code/syntax-highlight/_code.clarity.scss
-$clr-monoFont: Consolas, Monaco, Courier, monospace !default;
```

Comment:
fyi - those usage comments were from the dark theme implementation so I could figure out where and what things were being variables were used in our codebase. My concern here is for the small minority of folks who build their apps and compile clarity scss instead of just consuming it.